packages/vue-app/template/App.js

```diff
@@ -203,6 +196,20 @@ export default {
           )
         }
         <% } %>
+
+        <% if (features.fetch) { %>
+        // Wait for asyncData & old fetch to finish
+        await Promise.all(p)
```

Comment:
Does it make sense to empty all resolved promises after this line to cleanup refs?

Reply:
The performance point is memory management. Promises are stored as link lists. As fast as you free them up, garbage collector can empty the dependencies from memory. Still not huge only a nitpick :)

Reply:
Ah, makes sense! Learned something new today. Should be addressed in the most recent commit. Let me know if this can be resolved or if you have any other improvements/suggestions etc. :)